src/custom/pages/Profile/index.tsx

```diff
@@ -105,25 +91,29 @@ export default function Profile({ ENSName }: AccountDetailsProps) {
                 <span role="img" aria-label="wingedmoney">
                   💸
                 </span>
-                <strong>-</strong>
+                <strong>{profileData?.totalReferrals?.toLocaleString() || '-'}</strong>
                 <span>Total trades</span>
               </FlexCol>
               <FlexCol>
                 <span role="img" aria-label="handshake">
                   🤝
                 </span>
-                <strong>-</strong>
+                <strong>{format(profileData?.referralVolumeUsd)}</strong>
                 <span>Referrals Volume</span>
               </FlexCol>
             </FlexWrap>
           </ChildWrapper>
         </GridWrap>
         {!account && (
           <FlexWrap>
-            <Web3Status />
+            <Web3Status openOrdersPanel={() => console.log('TODO')} />
           </FlexWrap>
         )}
       </GridWrap>
     </Wrapper>
   )
 }
+
+const format = (number?: number): string => {
+  return number ? numberFormatter.format(number) : '-'
+}
```

Comment:
The volume amounts are supposed to come as strings, even though they aren't yet.
So, this fn might not be needed at all.

Reply:
Got it, will update it when the API is finalized.